blocks/columns/columns.css

```diff
@@ -224,7 +275,7 @@
     margin: 0;
 }
 
-.news .columns.quarter-circle img {
+.section:not(.leadspace) .image-end picture, .section:not(.leadspace) .image-start picture {
```

Comment:
Does the columns block have to know about leadspace? Would the following selector achieve the same result?

```
.columns.quarter-circle picture
```

or are there cases where image-start/end are used without quarter-circle? Then how about this?

```
.columns.image-start picture,
.columns.image-end picture {
```